topics/proteomics/tutorials/proteogenomics-novel-peptide-analysis/tutorial.md

```diff
@@ -164,68 +164,81 @@ The Multiomics Visualization Platform is a galaxy visualization plugin that allo
 >
 > 1) Click on the **Visualize in MVP application**, it will open up options for visualization application in the center pane, Select **MVP Application** from the options (or Right click to open in a new window).
 >
-> ![mz to sqlite](../../images/mz2sqlite.png){:width="20%"}
+> ![mz to sqlite](../../images/visualize.png){:width="20%"}
+> 
+> This will open in the Center Pane. Select the MVP application.
 >
-> 2) Click on **Load from Galaxy**.
+>![MVP](../../images/Open_MVP.png){:width="20%"}
+>
+> This is how it will look.
+>![Overview](../../images/Peptide_overview.png){:width="60%"}
+>
+> 2) Click on **Load from Galaxy** to open the list of peptides you would like to view.
+>
+> ![load from galaxy](../../images/load_from_Galaxy.png){:width="80%"}
+>
+>This will open a dropdown list. Select the novel peptides from there.
+>
+>
+> ![dropdown](../../images/DropDown_novel.png){:width="50%"}
 >
-> ![load from galaxy](../../images/load_from_Galaxy.png){:width="50%"}
 >
 > 3) Select **Novel Peptides** from the right hand side.
 >
-> ![novel peptides view](../../images/novel_peptides_view.png){:width="50%"}
+> ![novel peptides view](../../images/novel_peptides_view.png){:width="70%"}
 >
-> 4) Select any peptide, For eg: `ESSREALVEPTSESPRPALAR`, and then click on **Selected Peptide PSMs**.
+> 4) Select any peptide, For eg: `DGDLENPVLYSGAV`, and then click on **Selected Peptide PSMs**.
 >
-> ![select_pep_PSM](../../images/select_pep_PSM.png){:width="50%"}
+> ![select_pep](../../images/Filtering_novelpeptides.png){:width="70%"}
```

Comment:
don't forget to add these new images to the PR as well (and maybe remove old ones if they are no longer used?)